code/modules/admin/verbs/playsound.dm

```diff
@@ -52,6 +52,19 @@
 	playsound(get_turf(src.mob), S, 50, FALSE, FALSE)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Play Local Sound") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/play_direct_mob_sound(S as sound, mob/M)
+	set category = "Fun"
+	set name = "Play Direct Mob Sound"
+	if(!check_rights(R_SOUND))
+		return
+
+	if(!M)
+		M = input(usr, "Choose a mob to play the sound to. Only they will hear it.", "Play Mob Sound") as null|anything in sortNames(GLOB.player_list)
+	log_admin("[key_name(src)] played a direct mob sound [S] to [M].")

```

Comment:
Check if M isn't null before proceeding.

Reply:
Is that fine?